lib/Zonemaster/Backend/Config.pm

```diff
@@ -235,12 +239,56 @@ sub new_DB {
     $dbclass->import();
     $log->notice("Constructing database adapter: $dbclass");
 
-    my $db = $dbclass->new;
+    my $db = $dbclass->new({ config => $self });
 
     # Connect or die
     $db->dbh;
 
     return $db;
 }
 
+sub new_PM {
+    my $self = shift;
```

Comment:
This method is not documented. What is "PM"?

Reply:
PM is short for processing manager. I added documentation.